src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -403,4 +403,7 @@
   <data name="NoRoutesMatchedForPage" xml:space="preserve">
     <value>No page named '{0}' matches the supplied values.</value>
   </data>
+  <data name="UrlHelper_RelativePagePathIsNotSupported" xml:space="preserve">
+    <value>The relative page path '{0}' can only be from the context of executing a Razor page. Specify a root relative path with a leading '/' to calculate it in the current context.</value>
```

Comment:
`page path` -> `page name`
`can only be from the context of executing a Razor page` -> `can only be used while executing a Razor Page`.
`to calculate it in the current context` -> `to generate a URL outside of a Razor Page`